### ISSUE TITLE
CORS is failing with new API

### ISSUE THREAD

**Issue by mikkokotila** [Lotus-King-Research/Padma-Backend#80]
The current config is not working:

```
import uvicorn
from fastapi import FastAPI
from fastapi.middleware.cors import CORSMiddleware

app = FastAPI()

app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_headers=["*"],
)
```

A discussion about the topic [here](https://github.com/tiangolo/fastapi/issues/1663).

**Comment by mikkokotila** [comment]
Looks like there is an explicit port requirement. Without port will not work.

**Comment by mikkokotila** [comment]
Fixed with #79
